src/sdk/pynni/nni/parameter_expressions.py

```diff
@@ -57,7 +57,7 @@ def quniform(low, high, q, random_state):
     q: sample step
     random_state: an object of numpy.random.RandomState
```

Comment:
need to change here:
https://github.com/microsoft/nni/blob/master/src/sdk/pynni/nni/smartparam.py

Reply:
updated